capture_image.py

```diff
@@ -0,0 +1,50 @@
+import math
+import cv2
+import numpy as np
+import subprocess
+import imghdr
+import traceback
+import os
+from math import sin, cos
+from operator import add
+
+
+# finds rotation and translation of vision targets
+class CaptureImage:
+
+    # initilaze variables
+    def __init__(self, detector):
+
+        self.detector = detector
```

Comment:
If we change this constructor to just take a generator, then we can pass in a VideoLiveGenerator without having to import all the detector code.